projects/aca-content/src/lib/components/search/search-input.service.ts

```diff
@@ -23,17 +23,33 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Component, Input } from '@angular/core';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
 
-/**
- * This wrapper is designated to be used with 'adf-dynamic-component'.
- * It forwards the dynamic component inputs to original sidenav.
- */
-@Component({
-  selector: 'aca-sidenav-wrapper',
-  templateUrl: './sidenav-wrapper.component.html'
+@Injectable({
+  providedIn: 'root'
 })
-export class SidenavWrapperComponent {
-  @Input()
-  data: { mode?: 'collapsed' | 'expanded' } = {};
+export class SearchInputService {
```

Comment:
Also name of this service is a bit confusing since it has nothing to do with search input, it just manages the routes and navigates to search

Reply:
yeah, I will rename it to `SearchNavigationService` and maybe in the future should consider getting rid of it